### DISCUSSION TITLE
Newton Scheme FAILED to converge - Following up

### DISCUSSION THREAD

**Discussion by MonikaCarvajal** [idaholab/moose#16120] [Q&A General] [answered]
Hi all,
This is a follow up of the [this](https://groups.google.com/forum/?nomobile=true#!topic/moose-users/6NVngaMpYEo) topic on MOOSE-users.  
I get an error during simulation apparently something fails during the mesh adaptivity. I have a rectangular mesh generated with the built-in tool of MOOSE. 
My simulation on debug mode

_Building mesh ............................................................................
..........................................................................................
..........................................................................................
..........................................................................................
..........................................................................................
..........................................................................................
................................                                                           [559.82 s]

*** Warning, This code is deprecated and will be removed in future versions:
Use set() method for non-const access

Initializing equation system .............................................................
..............                                                                             [ 76.19 s]
Using EXPERIMENTAL Stateful Material Property projection with Adaptivity!

Warning! Mesh re-partitioning is disabled while using stateful material properties!  This can lead to large load imbalances and degraded performance!!

Framework Information:
MOOSE Version:           git commit 3a7a42ac3e on 2020-10-29
LibMesh Version:         a62cd9cef65556bd080c8fbed00a43016a2d5bff
PETSc Version:           3.13.3
SLEPc Version:           3.13.3
Current Time:            Fri Nov  6 11:43:37 2020
Executable Timestamp:    Fri Nov  6 11:27:31 2020

Parallelism:
  Num Processors:          4
  Num Threads:             1

Mesh: 
  Parallel Type:           replicated
  Mesh Dimension:          2
  Spatial Dimension:       2
  Nodes:                   
    Total:                 44800
    Local:                 11322
  Elems:                   
    Total:                 44341
    Local:                 11085
  Num Subdomains:          1
  Num Partitions:          4
  Partitioner:             metis

Nonlinear System:
  Num DOFs:                134400
  Num Local DOFs:          33966
  Variables:               { "disp_x" "disp_y" "porepressure" } 
  Finite Element Types:    "LAGRANGE" 
  Approximation Orders:    "FIRST" 

Auxiliary System:
  Num DOFs:                842479
  Num Local DOFs:          210615
  Variables:               { "stress_zx" "stress_zy" "stress_zz" "indicator" 
                             "marker" } 
  Finite Element Types:    "MONOMIAL" 
  Approximation Orders:    "CONSTANT" 

Execution Information:
  Executioner:             Transient
  TimeStepper:             FunctionDT
  Solver Mode:             NEWTON
  PETSc Preconditioner:    lu 
  MOOSE Preconditioner:    SMP_

**and the error logs**

Time Step 45, time = 0.1, dt = 0.00150267
Computing initial residual ...............................................................
..........................................................................................
..........                                                                                 [164.37 s]
Updating displaced mesh ...................                                                [ 20.57 s]
 0 Nonlinear |R| = 5.252897e+05
Updating displaced mesh ...................                                                [ 20.60 s]
      0 Linear |R| = 5.252897e+05
      1 Linear |R| = 7.326103e-07
  Linear solve converged due to CONVERGED_RTOL iterations 1
Updating displaced mesh ...................                                                [ 20.61 s]
 1 Nonlinear |R| = 2.330332e-05
Nonlinear solve converged due to CONVERGED_FNORM_ABS iterations 1
 Solve Converged!
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 11 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(1.18837e-06, -1.47947e-07,        0)   p=(x,y,z)=(      -1, -0.577351,        0)   error=1.19754e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 12 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(-1.18837e-06, 1.47947e-07,        0)   p=(x,y,z)=(      -1, -0.57735,        0)   error=1.19754e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 13 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(1.18837e-06,        0,        0)   p=(x,y,z)=(      -1, -0.57735,        0)   error=1.18837e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 14 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(-1.18837e-06, -1.47947e-07,        0)   p=(x,y,z)=(      -1, -0.577351,        0)   error=1.19754e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 15 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(1.18837e-06, 1.47947e-07,        0)   p=(x,y,z)=(      -1, -0.57735,        0)   error=1.19754e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 16 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(-1.18837e-06,        0,        0)   p=(x,y,z)=(      -1, -0.57735,        0)   error=1.18837e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 17 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(1.18837e-06, -1.47947e-07,        0)   p=(x,y,z)=(      -1, -0.577351,        0)   error=1.19754e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 18 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(-1.18837e-06, 1.47947e-07,        0)   p=(x,y,z)=(      -1, -0.57735,        0)   error=1.19754e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 19 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(1.18837e-06,        0,        0)   p=(x,y,z)=(      -1, -0.57735,        0)   error=1.18837e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 20 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(-1.18837e-06, -1.47947e-07,        0)   p=(x,y,z)=(      -1, -0.577351,        0)   error=1.19754e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
[0] ../src/fe/fe_map.C, line 1953, compiled Oct  7 2020 at 15:55:59
WARNING: Newton scheme has not converged in 21 iterations:
   physical_point=(x,y,z)=(  10.232, -1.91367,        0)   physical_guess=(x,y,z)=(  10.232, -1.91367,        0)   dp=(x,y,z)=(1.18837e-06, 1.47947e-07,        0)   p=(x,y,z)=(      -1, -0.57735,        0)   error=1.19754e-06   in element 681466
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
ERROR: Newton scheme FAILED to converge in 21 iterations in element 681466 for physical point = (x,y,z)=(  10.232, -1.91367,        0)
  Elem Information
   id()=681466, unique_id()=1272447, processor_id()=0
   type()=QUAD4
   dim()=2
   n_nodes()=4
    0  Node id()=590977, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449085) (0/1/449086) (0/2/449087) 
    1  Node id()=578621, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/439701) (0/1/439702) (0/2/439703) 
    2  Node id()=590980, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449094) (0/1/449095) (0/2/449096) 
    3  Node id()=590978, processor_id()=0, Point=(x,y,z)=(  10.232, -1.91367,        0)
    DoFs=(0/0/449088) (0/1/449089) (0/2/449090) 
   n_sides()=4
    neighbor(0)=681492
    neighbor(1)=666946
    neighbor(2)=681468
    neighbor(3)=681465
   hmin()=2.98957e-09, hmax()=4.23646e-09
   volume()=8.97373e-18
   active()=1, ancestor()=0, subactive()=0, has_children()=0
   parent()=666945
   level()=24, p_level()=0
   refinement_flag()=DO_NOTHING
   p_refinement_flag()=DO_NOTHING
   DoFs=(1/0/2501616) (1/1/2501617) (1/2/2501618) (1/3/2501619) (1/4/2501620) (1/5/2501621) (1/6/2501622) (1/7/2501623) (1/8/2501624) (1/9/2501625) (1/10/2501626) (1/11/2501627) (1/12/2501628) (1/13/2501629) (1/14/2501630) (1/15/2501631) (1/16/2501632) (1/17/2501633) (1/18/2501634) 
Exiting...
[0] ../src/fe/fe_map.C, line 1992, compiled Oct  7 2020 at 15:55:59
Outputting console ...........
Outlier Variable Residual Norms:
  porepressure: 2.330332e-05

Postprocessor Values:
+----------------+----------------+
| time           | dt             |
+----------------+----------------+
:                :                :
|   2.853117e-02 |   1.000000e-03 |
|   3.138428e-02 |   1.000000e-03 |
|   3.452271e-02 |   1.000000e-03 |
|   3.797498e-02 |   1.000000e-03 |
|   4.177248e-02 |   1.000000e-03 |
|   4.594973e-02 |   1.000000e-03 |
|   5.054470e-02 |   1.000000e-03 |
|   5.559917e-02 |   1.000000e-03 |
|   6.115909e-02 |   1.000000e-03 |
|   6.727500e-02 |   1.000000e-03 |
|   7.400250e-02 |   1.000000e-03 |
|   8.140275e-02 |   1.000000e-03 |
|   8.954302e-02 |   1.000000e-03 |
|   9.849733e-02 |   1.000000e-03 |
|   1.000000e-01 |   1.000000e-03 |
+----------------+----------------+

                                                             [ 12.52 s]
Outputting exodus ........................................................................
............................                                                               [101.15 s]
Adaptivity step 1 of 1 ...........Assertion `flags_were_consistent' failed.

Assertion `flags_were_consistent' failed.

Assertion `flags_were_consistent' failed.

Assertion `flags_were_consistent' failed.

[[2] ../src/mesh/mesh_refinement.C, line 536, compiled Oct  7 2020 at [3] ../src/mesh/mesh_refinement.C, line 536, compiled Oct  7 2020 at 15:56:30
0] ../src/mesh/mesh_refinement.C, line 536, compiled Oct  7 2020 at 15:56:30
[1] ../src/mesh/mesh_refinement.C, line 536, compiled Oct  7 2020 at 15:56:30
15:56:30
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 2
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 1
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 3

Thanks in advance!
Monika

**Comment by aeslaughter** [comment]
@fdkong or @roystgnr Can you offer some advice on this question?

**Comment by roystgnr** [comment, marked as answer]
The fe_map.C warnings are coming from inverse_map.  Typically that means a badly twisted element where the mapping Jacobian goes singular, but we do have an outstanding issue with pyramids that can be triggered near their apex vertex: https://github.com/libMesh/libmesh/pull/2749

The flags_were_consistent failure means that there's an element which has one refinement flag on one processor but a different refinement (or lack-of-refinement) flag on another processor.  If you're using a libMesh flagging routine then this is probably a bug in libMesh.  If you're using a homebrewed flagging routine that you're certain is correct on local elements, you can call MeshRefinement::make_flags_parallel_consistent() to sync your flags onto ghost elements.

**Comment by fdkong** [comment]
`MeshRefinement::make_flags_parallel_consistent() ` is called only when distributed mesh is used. For replicated mesh, do we need that? For replicated mesh, we mark flags for all elements redundantly in parallel. That could generate inconsistency?

It might be a good idea to fix the first issue (inverted elements) before we dig into the second one.

**Reply by roystgnr** [reply]
> For replicated mesh, do we need that?

Whether you need make_flags_parallel_consistent() is completely independent of whether your mesh is replicated or not.  You could write flags only to the local portion of a replicated mesh, and so then need make_flags_parallel_consistent() to sync the ghost flags.  Or you could use a parallel-consistent algorithm to write flags to local and ghost flags of a distributed mesh, after which no further sync would be required.

> For replicated mesh, we mark flags for all elements redundantly in parallel. That could generate inconsistency?

It depends on how you mark flags.  Elem::set_refinement_flag() doesn't do any parallel communication; if you call it with one flag on one processor and another flag on a different processor, we can't stop you; we can't even find that out until (in dbg mode) you hit the above failed assert later on.

> It might be a good idea to fix the first issue (inverted elements) before we dig into the second one.

Always a good heuristic.  In this case, if you're calculating data for flagging purposes based on local copies of element geometry, and the geometry is differing from processor to processor, then fix the geometry and the flagging would fix itself.  If that's not what's going on, well, you probably want to fix the geometry anyways (I just caught an IGA bug last month from an inverse_map warning about 2e-6 error...), so you might as well do that first and then look at the flagging afterwards.

**Comment by MonikaCarvajal** [comment]
Thanks for the replies. So it looks like comes from a bug... Can you advice on how to proceed?

**Reply by fdkong** [reply]
You could use ParaView or other visualization tools to look into the mesh after the first time step. Check if there are any invalid elements.
